src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -792,7 +792,7 @@ namespace SystemAvailabilityManager {
 				SetupOutputVariable( "Availability Manager Optimum Start Control Status []", OptStartSysAvailMgrData( SysAvailNum ).AvailStatus, "System", "Average", OptStartSysAvailMgrData( SysAvailNum ).Name );
 
 				// add
-				SetupOutputVariable( "Availability Manager Optimum Start Hours Before Occupancy []", OptStartSysAvailMgrData( SysAvailNum ).NumHoursBeforeOccupancy, "System", "Average", OptStartSysAvailMgrData( SysAvailNum ).Name, "Daily" );
```

Comment:
Well this is definitely an example of the kind of changes the sneak in on us and break things. In this case I can pretty confidently say that OpenStudio is not relying on this variable directly, but clients of either EnergyPlus or OpenStudio could have some dependency coded into software. I think it might not be the best idea to change a variable in bug fix.